contracts/implement/ConcentratedLiquidity.sol

```diff
@@ -98,10 +98,35 @@ abstract contract ConcentratedLiquidity {
         uint256 _amount
         ) internal virtual {}
 
+    // 1. Burn NFT
+    // 2. Update liquidity data
+    // 3. Transfer assets
+    // 4. Get fee reward
+    // 5. transfer fee reward
     function removeLiquidity(uint256 tokenId) public virtual {
         address owner = _msgSender();
         require(owner == positionDexNft.ownerOf(tokenId), "!Owner");
-        // TODO removeLiquidity and get fee reward
+
+        Liquidity.Data memory liquidityData = concentratedLiquidity[tokenId];
+
+        positionDexNft.burn(tokenId);
+        delete concentratedLiquidity[tokenId];
+
+        withdrawLiquidity(
+            liquidityData.pool,
+            owner,
+            SpotHouseStorage.Asset.Base,
+            uint256(liquidityData.baseVirtual)
```

Comment:
MUST call remove liquidity in pool, to get baseOut, quoteOut, after that transfer

DO NOT transfer with data of `liquidityData`